src/java/com/phenix/pct/PCTRun.java

```diff
@@ -229,6 +229,10 @@ public void setFailOnError(boolean failOnError) {
         runAttributes.setFailOnError(failOnError);
     }
 
+    public void setQuickRequest(boolean quickRequest) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1161)